image_services/egis/publishing_MDCS_to_EGIS.md

```diff
@@ -0,0 +1,75 @@
+# Publishing MDCS-generated Mosaic Datasets as Image Services on Earthdata GIS
+
+*This workflow is done in ArcGIS Pro 3.1*
+
+## Connect to NASA SIT ArcGIS Portal
+Publishers must add the NASA SIT Portal to their list of portals in ArcGIS. This only needs to be done once.
+
+1. Open a project in ArcGIS Pro using the Map template
+2. Click the Project menu and click on the Portals section
+3. Click the Add Portal button
+4. Enter the URL to the desired NASA portal, with /portal at the end (i.e. https://url_to_gis_server/portal/)
+5. Click OK
+6. Click on the options button for the portal, and choose to sign in. You will need to be on the VPN to work with this portal, including signing in.
+
+**It's easiest to connect to the NASA VPN first before creating the ArcGIS Project you'll be using for publishing, and always connect to the VPN before opening that project going forward when the active portal is set to the SIT environment.**
+
+*The same workflow is used for publishing to the NASA UAT Portal; simply connect to that portal in ArcGIS Pro rather than the NASA SIT Portal.*
+
+## Verify that the Mosaic Dataset paths are correct
+
+1. Navigate to the geodatabase with the mosaic dataset in the Catalog pane in ArcGIS Pro
+2. Expand the geodatabase to view the mosaic dataset
+3. Drag the mosaic dataset into the map view and verify that it behaves as expected
+   1. In many cases, the overviews will display correctly and the attribute table contains records for all of the source rasters, but the source rasters do not render. Zoom in to a single source raster, and make sure that it renders at all zoom levels.
+   2. If the source rasters render appropriately, skip the next step and proceed to editing the metadata
+
+## Repair Mosaic Dataset Pixel Cache Path (IF NECESSARY)
+
+If the source rasters do not render correctly, try repairing the path to the pixel cache in the mosaic dataset.
+
+1. Right-click on the Mosaic Dataset in the Table of Contents and select Data > Repair Mosaic Dataset Paths
+![Repair Paths](images/RepairPaths.PNG)
+2. Scroll to the path to the PixelCache directory, and replace it with a path on your local computer.
```

Comment:
```suggestion
2. Scroll to the path to the PixelCache directory, and replace it with any local path on your local computer. 
```
I think specifying that it can be any local path and/or giving an example path might be helpful. I can foresee myself getting confused by which path is right.

Reply:
This section needs some serious reworking to deal with the different paths impacted by the pixel cache issue; I will address this issue as part of that effort.

Reply:
Addressed in commit https://github.com/ASFHyP3/gis-services/pull/51/commits/0045ff9d7494c7904f234f7bebebad00eae486e1

(@jacquelynsmale more detail than you could have imagined possible....)